ranger-util/src/test/LICENSE.txt

```diff
@@ -0,0 +1,323 @@
+                                 Apache License
```

Comment:
Maven complained that this directory does not exist. This file can enforce that while not raising an alert with the rats build plugin.